labman/db/configuration_manager.py

```diff
@@ -58,6 +59,10 @@ def create(config_fp, test_env, db_host, db_port, db_name, db_user,
             Path to the configuration file
         test_env : bool
             If true, a config file for a test environment will be created
+        certificate_file : str
+            The certificate file for HTTPS
+        key_file : str
```

Comment:
I don't know if labman has a convention about this, but I really dislike naming variables "_file" that actually hold file *paths* (rather than python file objects).  I would rather see these named with a suffix like "_filepath" or "_fp".